packages/KPrompt/KPrompt.vue

```diff
@@ -0,0 +1,224 @@
+<template>
+  <KModal
+    :is-visible="isVisible"
+    :title="displayTitle"
+    class="k-prompt"
+    @keyup.esc="close">
+    <template v-slot:header-content>
+      <div class="k-prompt-header w-100">
+        <div class="k-prompt-header-content d-flex w-100">
+          <slot name="header-content">
+            <KIcon
+              v-if="type === 'warning'"
+              icon="warning"
+              color="var(--white)"
+              secondary-color="var(--yellow-400)"
+              size="20"
+              class="mr-2" />
+            {{ displayTitle }}
+          </slot>
+          <div class="close-button">
+            <KButton
+              class="non-visual-button"
+              @click="close">
+              <KIcon
+                icon="close"
+                color="var(--grey-600)"
+                size="15" />
+            </KButton>
+          </div>
+        </div>
+        <hr class="divider">
+      </div>
+    </template>
+    <template v-slot:body-content>
+      <div class="k-prompt-body">
+        <div class="k-prompt-body-content">
+          <slot name="body-content">
+            {{ message }}
+          </slot>
+
+          <div
+            v-if="confirmationText"
+            class="k-prompt-confirm-text w-100">
+            Type "<span class="bold-600">{{ confirmationText }}</span>" to confirm your action.
+
+            <KInput
+              v-model.trim="confirmationInput"
+              class="pt-2" />
+          </div>
+        </div>
+        <hr class="divider">
+      </div>
+    </template>
+    <template v-slot:footer-content>
+      <div class="k-prompt-action-buttons">
+        <slot name="action-buttons">
+          <KButton
+            appearance="outline"
+            class="k-prompt-cancel mr-2"
+            @click="close"
+            @keyup.esc="close">{{ cancelButtonText }}</KButton>
+          <KButton
+            :appearance="type === 'danger' ? 'danger' : 'primary'"
+            :disabled="disableProceedButton"
+            class="k-prompt-proceed"
+            @keyup.enter="proceed"
```

Comment:
Pressing `Enter` on the docs page does not trigger the `proceed` action or close the prompt for me.

Reply:
Fixed, but requires you to click on the dialog first.